.travis/travis_create_slaves.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -x -e
+
+if [ -z "$PG_VERSION" ]
+then
+    echo "env PG_VERSION not define";
+elif [ "x${PG_VERSION}" = "xHEAD" ]
+then
+    PG_CTL="/usr/local/pgsql/bin/pg_ctl"
+    PG_BASEBACKUP="/usr/local/pgsql/bin/pg_basebackup"
+else 
+    PG_CTL="/usr/lib/postgresql/${PG_VERSION}/bin/pg_ctl"
+    PG_BASEBACKUP="/usr/lib/postgresql/${PG_VERSION}/bin/pg_basebackup"
+fi
+
+#Create Slave 1
+sudo rm -rf ${PG_SLAVE1_DATADIR}
+sudo mkdir -p ${PG_SLAVE1_DATADIR}
+sudo chmod 700 ${PG_SLAVE1_DATADIR}
+sudo chown -R postgres:postgres ${PG_SLAVE1_DATADIR}
+
+sudo su postgres -c "$PG_BASEBACKUP -Upostgres -D ${PG_SLAVE1_DATADIR} -X stream"
+
+sudo su postgres -c "echo standby_mode = \'on\' >${PG_SLAVE1_DATADIR}/recovery.conf"
+sudo su postgres -c "echo primary_conninfo = \'host=localhost port=5432 user=test password=test\' >>${PG_SLAVE1_DATADIR}/recovery.conf"
+sudo su postgres -c "echo recovery_target_timeline = \'latest\' >>${PG_SLAVE1_DATADIR}/recovery.conf"
+
+if [ "x${PG_VERSION}" <> "xHEAD" ]
```

Comment:
Does this work?
The code gets executed for HEAD as well: https://travis-ci.org/pgjdbc/pgjdbc/jobs/307997910#L5427

Would you please try use `[[ .. ]]`?

Reply:
@vlsi I had fix this problem, Please check again.